src/components/Tooltip/TooltipWrapper.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Tooltip from "./Tooltip";
+
+const TooltipWrapper = ({ children, wrapWithTooltip, tooltipProps }) => {
+  if (!wrapWithTooltip) {
```

Comment:
not really need a wrapper for this. you can just set the component according to condition and set all the props which related to tooltip only if this condition apply

Reply:
Does this look better?
<img width="545" alt="image" src="https://user-images.githubusercontent.com/104433616/185791273-963b152c-9167-460e-a030-30bbb0295ee6.png">


I've made it in analogy with ClickableWrapper, which I've found rather useful, thought it's not necessary at all

Reply:
I like this approach more  :)

Reply:
Done :)